### ISSUE TITLE
Accuracy data on individual subjects in the test set

### ISSUE THREAD

**Issue by HarshavardhanaTG** [facebookresearch/emg2qwerty#65]
Is it possible to share accuracy data of 8 individual subjects in the test set (in the paper you report average and SD)? I am having some trouble with users 0, 6, and 7.

**Comment by HarshavardhanaTG** [comment]
Follow up. 

1) For values reported in ``Personalized (random-init)'' in table 3, are the accuracy vales of each subject averaged over multiple random seed? 

2) For values reported in ``Personalized (finetuned) '' in table 3, did accuracy values for all 8 individuals improve when initialized with weights from the generic model?

**Comment by viswanathgs** [comment]
Hi @HarshavardhanaTG,

> Is it possible to share accuracy data of 8 individual subjects in the test set (in the paper you report average and SD)? I am having some trouble with users 0, 6, and 7.

the individual scores for each of the 8 test subjects for each benchmark is listed in https://github.com/facebookresearch/emg2qwerty/blob/main/scripts/experimental_results.py#L14.

> For values reported in ``Personalized (random-init)'' in table 3, are the accuracy vales of each subject averaged over multiple random seed?

They are not, we simply aggregate over the 8 test users but each user's CER is not aggregated over random seeds

> For values reported in ``Personalized (finetuned) '' in table 3, did accuracy values for all 8 individuals improve when initialized with weights from the generic model?

https://github.com/facebookresearch/emg2qwerty/blob/main/scripts/experimental_results.py#L14 has the full breakdown without aggregation for each of the benchmarks, and this should hopefully answer your question. Thanks!

Please re-open this issue if I can be of any further help. Thank you for your interest :)

**Comment by HarshavardhanaTG** [comment]
Thanks a lot. 

One quick question: if the CER for each individual was not averaged over multiple random seeds in ``Personalized (random-init)'' before comparing with the fine-tuned model, how do you conclude that ``Personalized (finetuned) ''  improves performance, given training variance?

Also, how was p < 0.005 computed in that case? Wouldn't averaging CER across many random seeds per subject be necessary for a reliable statistical test?

Thanks.
